recipe/meta.yaml

```diff
@@ -228,9 +228,9 @@ outputs:
         - {{ pin_subpackage("libclang" ~ libclang_soversion, max_pin=None) }}
       ignore_run_exports_from:
         # the build fails if it doesn't find the following, but it's not used
-        - libxml2  # [unix]
-        - zlib
-        - zstd
+        - libxml2
+        - zlib     # [win]
+        - zstd     # [win]
```

Comment:
This doesn't seem correct to me.

Looking at the logs on main (newer ones deleted or not done yet):
```
WARNING (libclang13,Library/bin/libclang-13.dll): Needed DSO zlib.dll found in ['libzlib']
WARNING (libclang13,Library/bin/libclang-13.dll): .. but ['libzlib'] not in reqs/run, (i.e. it is overlinking) (likely) or a missing dependency (less likely)
WARNING (libclang13,Library/bin/libclang-13.dll): Needed DSO Library/bin/zstd.dll found in ['zstd']
WARNING (libclang13,Library/bin/libclang-13.dll): .. but ['zstd'] not in reqs/run, (i.e. it is overlinking) (likely) or a missing dependency (less likely)
```

Since the clang host dep isn't transitive, I don't see how ignoring these run exports on windows can lead to the right outcome.